docs/source/configuration_file.rst

```diff
@@ -932,27 +932,33 @@ being used for the segmentation task).
            "dropout_rate": {
                "type": "float"
            },
-           "batch_norm_momentum": {
-               "type": "float"
+           "bn_momentum": {
+               "type": "float",
+               "$$description": [
+                    "Defines the importance of the running average: (1 - `bn_momentum`). A large running\n",
```

Comment:
The list of strings is not very convenient. @ahill187 would there be a way to automatically set the width of the column so we could simply enter a string in the description field instead of needing to manually assess where we want our new lines?

Reply:
@andreanne-lemay ah yes this is quite annoying haha. The current implementation of the package `sphinx-jsonschema` doesn't seem to allow for that, however I could see about writing a patch that fixes it?

Reply:
that would be great! But this is not an urgent matter, I can open an issue about this.

Reply:
Sounds good!